internal/handlers/excel_handler.go

```
@@ -0,0 +1,86 @@
+            "error":   err.Error(),
+        })
+        return
+    }
+    
+    c.JSON(http.StatusOK, result)
+}
+
+// GET /api/v1/process/excel/local - 테스트용 api
+func (h *ExcelHandler) ProcessLocalExcel(c *gin.Context) {
+    filePath := "uploads/2025_6_1_ko.xlsx" 
+    cwd, err := os.Getwd()
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)